restcomm/restcomm.application/src/main/webapp/WEB-INF/conf/restcomm.xml

```diff
@@ -670,6 +670,11 @@
 					If no response received for 3 consecutive requests, connection will be killed
 					and attempted to connect again -->
 				<enquirelinkdelay>30000</enquirelinkdelay>
+				<!-- when no values are set, default value from code is UTF-8-->
+				<!--
+				<inboundencoding>UTF-8</inboundencoding>
```

Comment:
i thought we mentioned to use whatever hardcoded values we had,so to prevent breakingChanges. I think outbound was GSM7,right?

Reply:
we can set UTF8 just for stagign/cloud using the autoconfig scripts, and docker conf

Reply:
modified `restcomm.xml` and the `Smpp` constants